packages/fxa-auth-db-mysql/lib/db/mysql.js

```diff
@@ -1364,6 +1364,16 @@ module.exports = function (log, error) {
       )
   }
 
+  const READ_SECURITY_EVENTS = 'CALL readSecurityEvents_1(?)'
```

Comment:
The distinction between `READ_SECURITY_EVENTS` and `FETCH_SECURITY_EVENTS` or `readSecurityEvents` and `fetchSecurityEvents` isn't really clear. We could rename `fetchSecurityEvents_1` for clarity, but it might be easier to call `readSecurityEvents_1` to something like `fetchSecurityEventsByUid_1`. Then at least the two are obviously related, but the new one says what it's fetching by.